apps/airdrop/src/App/Buttons/GithubButton.tsx

```diff
@@ -0,0 +1,27 @@
+import { ActionButton } from "@namada/components";
```

Comment:
Not in this file, but could remove `apps/airdrop/src/App/Buttons/CosmosButton.tsx` as it is empty

Reply:
I think they are used atm! :)